.github/workflows/ci.yml

```diff
@@ -11,20 +11,25 @@ on:
   # Always run on PRs
   pull_request:
 
+concurrency:
```

Comment:
Careful: have concurrency (regardless of cancel) might prevent:
 * publishing snapshots for ALL commits on main
 * Populating caches for all commits on main (i.e. a PR opened from any commit won't run cached CI)
 * Potentially others depending on side effects of the workflow (e.g. coverage/sonar/qodana baseline, which might not be applicable to this project)

If any of those is a thing you want to keep, consider mixing in `github.sha` based on the value of github.event_name, so each commit on becomes it's own group for main and only pull_requests are cancelled.

Reply:
We use this in other projects and it works fairly well https://github.com/slackhq/circuit/blob/main/.github/workflows/ci.yml#L16

Reply:
Makes sense, up to you, just wanted to call it out, because we have had some very hard to notice problems with this (3rd point above). That repo was also higher throughput though.

A small note re the quoted string, Zac: `ci-` and `github.workflow` (=`CI`) are almost the same. Is there a reason for using both?

Reply:
just coincidence, it's not a big deal and I'm marking this resolved